src/collector.js

```diff
@@ -0,0 +1,53 @@
+import { default as Message } from 'message';
+
+// "I have a display case ready and waiting for our newest acquistions!"
+// --Taneleer Tivan
+
+
+export default class Collector {
+
+  constructor() {
+    this.errors = [];
+    this.notices = [];
+    this.warnings = [];
+  }
+
+  getMessageList(type) {
```

Comment:
In the tests you're accessing the `errors`, `notices`, and `warnings` properties all directly: https://github.com/mozilla/addons-validator/pull/35/files#diff-8f475bdab4471a61c3e772723fae0051R47

This feels like a cleaner/shorter/more obvious API, adding the `getMessageList` doesn't seem necessary.